vilebot/src/test/java/vilebot/handlers/user/DownOrJustMeTest.java

```diff
@@ -0,0 +1,110 @@
+package vilebot.handlers.user;
+
+import com.oldterns.vilebot.handlers.user.DownOrJustMe;
+import org.junit.Before;
+import org.junit.Test;
+import org.pircbotx.hooks.events.MessageEvent;
+
+import static org.mockito.Mockito.*;
```

Comment:
Not a blocker, but wildcard imports are not recommended.

Reply:
Fixed in 863a74f974bfeaf6799ee12a73f205f349259d7d.